src/sgame/sg_bot_local.h

```diff
@@ -131,6 +131,7 @@ struct botMemory_t
 	glm::vec3 stuckPosition;
 
 	int spawnTime;
+private:
```

Comment:
why?

Reply:
it's a value that has a setter function, it's arguably not meant to be public

Reply:
fair. lgtm